### PR TITLE
Update advanced CTA API documentation

### PR DESCRIPTION
**Pull request by samking**
We got a support ticket about our export API documentation for advanced CTA filters, and it was pretty out of date, so I updated it.  

I went through all of the filters that are possible on a CTA, reordered the keys to be in the same order we present in the UI, removed unavailable filters, and added documentation for available filters where documentation was incomplete.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,27 +184,26 @@ A call to action is something that folks are supposed to do (in particular, prof
...
-      "token": null, // this will be a deprecated field shortly
-      "role": {"organizer": false, "volunteer": true, "campaignDirector": true}, // only keys possible
-      "state": ["WI"], // list of states of the users who should see the CTA
-      "region": [3368], // list of regions of the users who should see the CTA
-      "zipCode": ["53201"], // list of zip codes to target users by
       "joinDate": {"type": "between", "toMts": 1600981504354, "fromMts": 1600549504354}, // currently, only the "between" type is supported, can not create multiple date filters, so this is just one dictionary
-      "listSize": {"max": 10, "min": 0, "type": "between"}, //  currently, only the "between" type is supported, only one possible set of entries here
+      "region": [3368], // list of regions of the users who should see the CTA
+      "role": {"campaignDirector": true, "organizer": false, "volunteer": true, "contact": false}, // only keys possible
       "assignedTo": ["fbei678"], // when targeting by a leader's parent, a list of those parents
```
Comment (sierrakn) [inline README.md]: Should this say, "a list of those parents' eids"

File: README.md [hunk 2]
```diff
@@ -184,27 +184,26 @@ A call to action is something that folks are supposed to do (in particular, prof
...
-      "outreachTask": null, // not currently implemented
-      "lastActiveDate": null, // not currently implemented
+      "listSize": {"max": 10, "min": 0, "type": "between"}, //  currently, only the "between" type is supported, only one possible set of entries here
```
Comment (sierrakn) [inline README.md]: Why is only the between type supported for this and the date filter?

**Review by sierrakn — APPROVED** [review]
LGTM, a couple questions

File: README.md (hunk 2 above)
Comment (samking) [reply README.md]: That was inaccurate for listSize.  I added an explanation of why the joined date filter is the way it is.

**Pull request merged**